decidim-core/db/migrate/20161020080756_add_position_to_steps.rb

```diff
@@ -0,0 +1,5 @@
+class AddPositionToSteps < ActiveRecord::Migration[5.0]
+  def change
+    add_column :decidim_participatory_process_steps, :position, :integer
```

Comment:
Right now, steps are created with `position: nil`, thus positioning themselves in the beginning of the list. I don't know if this is the expected or we'd rather have them in the end. If we want to have them in the end, I can create the steps with a really high position (`position: 9999` for example). Once the user reorders this, position will go back to a "normal" number